src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3066,14 +3066,20 @@ def _update_ssl_binding(cmd, resource_group_name, name, certificate_thumbprint,
     webapp_certs = client.certificates.list_by_resource_group(cert_resource_group_name)
 
     found_cert = None
+    # search for a cert that matches in the app service plan's RG
     for webapp_cert in webapp_certs:
         if webapp_cert.thumbprint == certificate_thumbprint:
             found_cert = webapp_cert
+    # search for a cert that matches in the webapp's RG
     if not found_cert:
         webapp_certs = client.certificates.list_by_resource_group(resource_group_name)
         for webapp_cert in webapp_certs:
             if webapp_cert.thumbprint == certificate_thumbprint:
                 found_cert = webapp_cert
+    # search for a cert that matches in the subscription, filtering on the serverfarm
+    if not found_cert:
+        sub_certs = client.certificates.list(filter=f"ServerFarmId eq '{webapp.server_farm_id}'", api_version="2021-03-01")
```

Comment:
> api_version="2021-03-01"

May I ask why do we need to hard code the api-version here?

Reply:
good catch, thanks. I didn't mean to leave that in

Reply:
Just removed that